tests/sentry/issues/endpoints/test_organization_group_search_views.py

```diff
@@ -1415,3 +1415,76 @@ def test_created_by_me_sort_by_popularity(self) -> None:
         assert response.data[0]["id"] == str(view_3.id), response.data[0]["starred"]
         # ============= Non-starred views =============
         # None
+
+    @with_feature({"organizations:issue-stream-custom-views": True})
+    @with_feature({"organizations:global-views": True})
+    def test_created_by_me_sort_by_created(self) -> None:
+        self.login_as(user=self.user_1)
+
+        view_1 = self.create_view(user=self.user_1, name="View 1")
+        view_2 = self.create_view(user=self.user_1, name="View 2")
+        view_3 = self.create_view(user=self.user_1, name="View 3")
+
+        response = self.client.get(self.url, {"createdBy": "me", "sort": "created"})
+        assert response.status_code == 200
+        assert len(response.data) == 3
+        # =============   Starred views   =============
+        # ============= Non-starred views =============
+        assert response.data[0]["id"] == str(view_1.id), not response.data[0]["starred"]
+        assert response.data[1]["id"] == str(view_2.id), not response.data[1]["starred"]
+        assert response.data[2]["id"] == str(view_3.id), not response.data[2]["starred"]
+        # None
```

Comment:
should we remove these None comments

Reply:
Yeah these were copied over from previous tests but don't need them here - removed these and other unnecessary comments/assertions